ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -80,20 +80,6 @@ impl<P: Parameters> GroupAffine<P> {
         self.mul_bits(cofactor)
     }
 
-    /// Multiplies `self` by the scalar represented by `bits`. `bits` must be a
-    /// big-endian bit-wise decomposition of the scalar.
-    pub(crate) fn mul_bits(&self, bits: impl Iterator<Item = bool>) -> GroupProjective<P> {
```

Comment:
I also don't understand what `pub(crate)` does, does it matter that it was removed in the new version of `mul_bits`?

Reply:
It makes `mul_bits` public outside the crate, when previously it was private.

Reply:
So it's no longer needed because now it's in the trait, not the impl?

Reply:
Yeah, by moving it into the trait it's publicly visible